calculator/README.md

```diff
@@ -1,4 +1,4 @@
-# AWESOME-BOOKS | MODULE-II
```

Comment:
There is an issue with your Github actions not triggering your linters checks. Please make the necessary changes in your files to fix this bug 📝 You could rely on this [link](https://github.com/microverseinc/linters-config/tree/master/react-redux) to find the issue 🤓 

One of the possible reasons could be the `/calculator` folder that you have in your main directory. I assume that linters are not getting triggered because their files are on a second level on the branch hierarchy